src/lib/assetUtils.ts

```diff
@@ -36,7 +36,7 @@ export const getDisplayableAssetFromTicker = (
  */
 export const getDisplayableTickerFromMarket = (market: string): string => {
   const [fullBaseAsset, quoteAsset] = market.split('-');
-  const base = getDisplayableAssetFromBaseAsset(fullBaseAsset);
+  const base = getDisplayableAssetFromBaseAsset(fullBaseAsset!);
```

Comment:
hm this might not be right since there is a check for `fullBaseAsset` on line 41

Reply:
It can never be null because even an empty string split is one element